geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/server/AbstractHitsMissesIntegrationTest.java

```diff
@@ -195,6 +195,13 @@ public void testPersist() {
     runCommandAndAssertNoStatUpdates("hash", (k) -> jedis.persist(k));
   }
 
+  /************* Sorted Set related commands *************/
+  @Test
+  public void testZIncrBy() {
+    runCommandAndAssertNoStatUpdates("key", 100.0,
+        (String k, Double i, String m) -> jedis.zincrby(k, i, m));
```

Comment:
I think we'll want to refer to an existing `"sortedset"` here.
Also, consider not creating a specialized `runCommandAssAssertNoStatUpdates` in favor of simply passing a fixed value in the lambda:
```
runCommandAndAssertNoStatUpdates("key", (k, v) -> jedis.zincrby(k, 100.0, v));
```

Reply:
already changed :)